.github/workflows/auth-post-merge.yaml

```
@@ -48,12 +53,21 @@ jobs:
         working-directory: ./auth
         run: sam validate --lint
 
-      - name: SAM build and test
+      - name: SAM build
         run: sam build
 
-      - name: Deploy SAM app
+      - name: Deploy Auth SAM app
         uses: govuk-one-login/devplatform-upload-action@v3.9.4
         with:
           artifact-bucket-name: ${{ secrets.ARTIFACT_BUCKET_NAME }}
           signing-profile-name: ${{ secrets.SIGNING_PROFILE_NAME }}
-          working-directory: ./auth/.aws-sam/build
\ No newline at end of file
+          working-directory: ./auth/.aws-sam/build
+
+      - name: Build, tag, and push testing images to Amazon ECR
+        uses: govuk-one-login/devplatform-upload-action-ecr@v1.3.0
+        with:
+          container-sign-kms-key-arn: ${{ secrets.CONTAINER_SIGN_KMS_KEY }}
```

```suggestion
          container-sign-kms-key-arn: ${{ vars.CONTAINER_SIGN_KMS_KEY }}
```
Might be worth using variables instead so we can see the values